avocado/plugins/runner_nrunner.py

```diff
@@ -177,7 +177,7 @@ async def _update_status(self, job):
                        for runtime_task in self.tasks}
         while True:
             try:
-                (task_id, status, _) = self.status_repo.status_journal_summary.pop()
+                (task_id, status, _) = self.status_repo.status_journal_summary.pop(0)
```

Comment:
I am just thinking here. what about use `status_journal_summary` as queue instead of list?

Reply:
Could be, but given that we're in "bug fix" only mode, I decided to do the most trivial fix instead.

Reply:
I think poping index 0 from the list makes it act like a queue. Or, are you proposing another structure @richtja?

Reply:
I'm ok with the current quick fix. :+1:

Reply:
I think I got what you meant @richtja, and I agree with it, but I'll go with @clebergnu proposal as a quick fix.

Reply:
I am ok with it as a quick fix, I just want to point it out that it is not efficient. And we shouldn't forget it in the future.

Reply:
> I am ok with it as a quick fix, I just want to point it out that it is not efficient. And we shouldn't forget it in the future.

Agreed! Issue please! :)

Reply:
ok I will do it